app/pages/BeelieversAuction/BeelieversBid.tsx

```diff
@@ -1,6 +1,5 @@
 import { FormProvider, useForm } from "react-hook-form";
 import { SUI } from "~/lib/denoms";
-import { AttemptAuction } from "./AttemptAuction";
 import { Card, CardContent } from "~/components/ui/card";
```

Comment:
The content of this PR is different then what the title suggests

Reply:
Forgot to push the other commit. Check now.